sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -125,9 +125,7 @@
   var err = gl.getError();
   if (err == gl.NO_ERROR) {
     testFailed("should NOT be able to create type " + formatName);
-  } else if (err == gl.INVALID_OPERATION) {
-    testFailed("should return gl.INVALID_ENUM for type " + formatName);
-  } else if (err == gl.INVALID_ENUM) {
+  } else if (err == gl.INVALID_ENUM || err == gl.INVALID_OPERATION) {
```

Comment:
Note that this extension is promoted to core and not available in ES3/WebGL 2.0, so the ES3 rules don't apply to this error check.